src/views/DogDetail.test.js

```diff
@@ -0,0 +1,9 @@
+import { render, screen } from '@testing-library/react';
+import Dog from './DogDetail';
+
+test('dog details rendering', () => {
+  const container = render(
```

Comment:
This test should be asnync with an await findByText() to get a proper snapshot and avoid a memory leak